Samples/SnowPal-Win-101/snowpal/MainWindow.xaml.cs

```diff
@@ -0,0 +1,125 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace snowpal
+{
+    public sealed partial class MainWindow : Window
+    {
+        // Public properties
+        public SnowpalViewModel ViewModel { get; }
```

Comment:
```suggestion
        public SnowpalViewModel ViewModel { get; } = new();
```
You can just initialize this here vs. needing the extra line in the constructor.